core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -161,10 +160,13 @@ class AdminClient(val time: Time,
     if (group.protocolType != ConsumerProtocol.PROTOCOL_TYPE)
       throw new IllegalArgumentException(s"Group ${groupId} with protocol type '${group.protocolType}' is not a valid consumer group")
 
-    group.members.map {
-      case member =>
-        val assignment = ConsumerProtocol.deserializeAssignment(ByteBuffer.wrap(member.assignment))
-        new ConsumerSummary(member.memberId, member.clientId, member.clientHost, assignment.partitions().asScala.toList)
+    if (group.state == "Stable") {
+      group.members.map { member =>
+          val assignment = ConsumerProtocol.deserializeAssignment(ByteBuffer.wrap(member.assignment))
```

Comment:
Thanks for addressing my comment. Unfortunately, the indenting is not right yet.